src/main/java/org/folio/rest/impl/VoucherNumberImpl.java

```diff
@@ -72,22 +73,24 @@ private void getVoucherNumber(Map<String, String> okapiHeaders, Handler<AsyncRes
     log.debug("getVoucherNumber:: Getting voucher number by query: {}", voucherNumberQuery);
     vertxContext.runOnContext((Void v) -> {
       String tenantId = TenantTool.calculateTenantId(okapiHeaders.get(RestVerticle.OKAPI_HEADER_TENANT));
-      PostgresClient.getInstance(vertxContext.owner(), tenantId).selectSingle(voucherNumberQuery, reply -> {
-        try {
-          if (reply.succeeded()) {
-            String voucherNumber = reply.result().getLong(0).toString();
-            log.info("getVoucherNumber:: Retrieved voucher number: {}", voucherNumber);
-            SequenceNumber sequenceNumber = new SequenceNumber().withSequenceNumber(voucherNumber);
-            asyncResultHandler.handle(buildOkResponse(sequenceNumber));
-          } else {
-            log.error("getVoucherNumber:: Failed to retrieve voucher number", reply.cause());
-            asyncResultHandler.handle(buildErrorResponse(reply.cause()));
-          }
-        } catch (Exception e) {
-          log.error("Error while handling response for voucher number request", e);
-          asyncResultHandler.handle(buildErrorResponse(e));
-        }
-      });
+      PostgresClient.getInstance(vertxContext.owner(), tenantId)
+        .selectSingle(String.format(voucherNumberQuery, tenantId),
```

Comment:
seems found issue here, we do not need to use String.format for VOUCHER_NUMBER_QUERY

Reply:
in this method VoucherNumberImpl::getVoucherStorageVoucherNumber

Reply:
indeed, I will fix this